app/models/jurisdiction.rb

```diff
@@ -194,6 +195,17 @@ def create_integration_mappings_async
     end
   end
 
+  def create_permit_type_required_steps
```

Comment:
Does this need a data migration for existing permit types?

Reply:
I can manually fix the existing ones on DEV/TEST. Seeded permit types will have this already